### PR TITLE
update: address data HTML

### PR DESCRIPTION
**Pull request by eneemann**
Added in a minor change for Bluff.

### CODE REVIEW

File: data/location/address-data/index.html [hunk 1]
```diff
@@ -34,7 +34,8 @@
   updates:
     - December 2022 - Added several address grids in Beaver county (Elk Meadows, Ponderosa,
       Greenville, Adamsville, Sulphurdale).  Made major updates to grids in Utah, Cache, Tooele,
-      and Box Elder Counties.  Renamed 'NSL' to 'North Salt Lake' and 'East Carbon City' to 'East Carbon'.
+      and Box Elder Counties.  Renamed 'NSL' to 'North Salt Lake' and 'East Carbon City' to 'East Carbon'.
```
Comment (steveoh) [inline data/location/address-data/index.html:37]: **suggestion**: I think you'll want this to be on the same line?

Comment (steveoh) [reply data/location/address-data/index.html:37]: actually it all looks kinda the same.

**Review by steveoh — APPROVED** [review]

File: data/location/address-data/index.html (hunk 1 above)
Comment (eneemann) [reply data/location/address-data/index.html:37]: Yeah, the long line/multiline stuff always throws me for a loop.  I put it on multiple lines for readability, but it seems like it renders the same anyways.

Comment (steveoh) [reply data/location/address-data/index.html:37]: yeah that is html for ya

**Pull request merged**